### DISCUSSION TITLE
Does Taffy support absolute positioning via `node.set{Top,Left,Right,Bottom}`?

### DISCUSSION THREAD

**Discussion by thecodrr** [DioxusLabs/taffy#600] [Q&A] [answered]
I see there is a `node.setPosition` API with `absolute` as an option but there's no way (that I can see) to actually set the top/right/bottom/left values. Is this supported at all? What's the best workaround?

**Comment by nicoburns** [comment, marked as answer]
Those values are namespaces under the (css standard) `inset` property. You can set them via the sub properties of inset.
